dummy_lambda/func/main.py

```diff
@@ -53,23 +53,26 @@ def return_foobar(**kwargs):
 def test_kwargs_passed_to_default_path_include_all(logger, event, **kwargs):
     try:
         assert kwargs.get("foo", None) == "bar"
-    except Exception:
-        raise FailButContinue("foo was not set to bar")
+    except AssertionError as e:
+        raise FailButContinue("foo was not set to bar") from e
 
 
 def test_kwargs_passed_to_default_path(foo, logger, event, **kwargs):
     try:
         assert foo == "bar"
-    except Exception:
-        raise FailButContinue("foo was not set to bar")
+    except AssertionError as e:
+        raise FailButContinue("foo was not set to bar") from e
 
 
 def throw_exception(**kwargs):
-    try:
-        raise Exception("Test event. Please ignore.")
-    except Exception as e:
-        sentry.capture(e)
-        raise FailButContinue from e
+    raise Exception()
+
+
+#     try:
```

Comment:
nit: should remove commented-out code from production-ready code.

Reply:
This is test code and [is excluded](https://github.com/mintel/lpipe/blob/v3.0.0/setup.py#L39) from the built package, but I do agree with the sentiment. In this specific case, I'm going to open another PR to re-activate these tests.